documentation/book/con-metrics-kafka-mini-kube-shift.adoc

```diff
@@ -0,0 +1,18 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-metrics-setup.adoc
+[id='con-metrics-kafka-mini-kube-shift-{context}']
+
+= Using metrics with `minikube` or `minishift`
+
+When adding Prometheus and Grafana servers to an Apache Kafka deployment using `minikube` or `minishift`, the memory available to the virtual machine should be increased (to 4 GB of RAM, for example, instead of the default 2 GB).
+
+For information on how to increase the default amount of memory, see:
+
+* xref:installing_kubernetes_cluster[]
+* xref:installing_openshift_cluster[].
+
+.Additional resources
+
+* https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/[Prometheus - Monitoring Docker Container Metrics using cAdvisor] describes how to use cAdvisor (short for container Advisor) metrics with Prometheus to analyze and expose resource usage (CPU, Memory, and Disk) and performance data from running containers within pods on Kubernetes.
+* https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md[cAdvisor Metrics] provides a full list of cAdvisor metrics as exposed through Prometheus.
```

Comment:
I'm not sure we need this link here.

Reply:
Is that because they'll never be used? I'll drop the link.